src/bundle/commitments.rs

```diff
@@ -187,3 +125,62 @@ pub(crate) fn hash_issue_bundle_auth_data(bundle: &IssueBundle<Signed>) -> Blake
     h.update(&<[u8; 64]>::from(bundle.authorization().signature()));
     h.finalize()
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{
+        builder::{Builder, BundleType},
+        bundle::commitments::hash_bundle_txid_data,
+        keys::{FullViewingKey, Scope, SpendingKey},
+        note::AssetBase,
+        orchard_flavor::OrchardVanilla,
+        value::NoteValue,
+        Anchor,
+    };
+    use rand::{rngs::StdRng, SeedableRng};
+
+    #[test]
+    fn test_hash_bundle_txid_data() {
```

Comment:
-Please rename to:
`fn test_hash_bundle_txid_data_for_orchard_vanilla()`
-Please add a comment about what are we testing here

Reply:
Done